components/brave_rewards/resources/shared/lib/locale_context.ts

```diff
@@ -5,7 +5,7 @@
 
 import * as React from 'react'
 
-export { default as formatMessage } from '$web-common/formatMessage'
+export { formatString } from '$web-common/formatString'
```

Comment:
Hey, I think maybe we shouldn't reexport this from here - people who look at Rewards might think they need to introduce their own string formatting utility